src/components/pnp/findStepColumns.ts

```diff
@@ -69,7 +69,7 @@ const chooseStep = (
 ): Step | undefined => {
   const alias = ApprovedAliases.get(label.toLowerCase());
   if (alias) {
-    return alias;
+    return available.has(alias) ? alias : undefined;
```

Comment:
I must not be reading this correctly.  How does this line do what the commit message states?

Reply:
The loop above that calls this method facilities the problem reporting & remaining steps. Is that what you mean? 
https://github.com/SeedCompany/cord-api-v3/blob/b83b214a2de85f1765c6259d362b609a72af99fc/src/components/pnp/findStepColumns.ts#L55-L61

Reply:
I got it now; for some reason I was kind of inverting it yesterday in my head.

Thanks!